packages/common/src/types/field.ts

```diff
@@ -234,7 +239,7 @@ export interface Field {
     compact?: CompactOrAlias;
     round?: number;
     format?: Format;
-    groupLabel?: string;
+    groups?: GroupType[];
```

Comment:
To avoid duplicating groups information across dimensions and metrics we can keep this as group references. In the FE we can find the group label and description from the table.

`type Field` -> `groups?: string[];`
`type TableBase`-> `groupDetails?: Record<string, GroupType>`

Reply:
Done as requested, thanks